test/functional/ui/cmdline_spec.lua

```diff
@@ -674,98 +547,73 @@ describe('external cmdline', function()
+      pos = 11,
+    }}, wildmenu_items=expected, wildmenu_pos=0}
 
     feed('<tab>')
-    screen:expect([[
+    screen:expect{grid=[[
       ^                         |
       {1:~                        }|
       {1:~                        }|
       {1:~                        }|
                                |
-    ]], nil, nil, function()
-      eq({{
-        content = { { {}, "sign jump"} },
-        firstc = ":",
-        indent = 0,
-        pos = 9,
-        prompt = ""
-      }}, cmdline)
-      eq(expected, wild_items)
-      eq(1, wild_selected)
-    end)
+    ]], cmdline={{
+      firstc = ":",
+      content = {{"sign jump"}},
+      pos = 9,
+    }}, wildmenu_items=expected, wildmenu_pos=1}
 
     feed('<left><left>')
-    screen:expect([[
+    screen:expect{grid=[[
       ^                         |
       {1:~                        }|
       {1:~                        }|
       {1:~                        }|
                                |
-    ]], nil, nil, function()
-      eq({{
-        content = { { {}, "sign "} },
-        firstc = ":",
-        indent = 0,
-        pos = 5,
-        prompt = ""
-      }}, cmdline)
-      eq(expected, wild_items)
-      eq(-1, wild_selected)
-    end)
+    ]], cmdline={{
+      firstc = ":",
+      content = {{"sign "}},
+      pos = 5,
+    }}, wildmenu_items=expected, wildmenu_pos=-1}
 
     feed('<right>')
-    screen:expect([[
+    screen:expect{grid=[[
       ^                         |
       {1:~                        }|
       {1:~                        }|
       {1:~                        }|
                                |
-    ]], nil, nil, function()
-      eq({{
-        content = { { {}, "sign define"} },
-        firstc = ":",
-        indent = 0,
-        pos = 11,
-        prompt = ""
-      }}, cmdline)
-      eq(expected, wild_items)
-      eq(0, wild_selected)
-    end)
+    ]], cmdline={{
+      firstc = ":",
+      content = {{"sign define"}},
+      pos = 11,
+    }}, wildmenu_items=expected, wildmenu_pos=0}
 
     feed('a')
-    screen:expect([[
+    screen:expect{grid=[[
       ^                         |
       {1:~                        }|
       {1:~                        }|
       {1:~                        }|
                                |
-    ]], nil, nil, function()
-      eq({{
-        content = { { {}, "sign definea"} },
-        firstc = ":",
-        indent = 0,
-        pos = 12,
-        prompt = ""
-      }}, cmdline)
-      eq(nil, wild_items)
-      eq(nil, wild_selected)
-    end)
+    ]], cmdline={{
+      firstc = ":",
+      content = {{"sign definea"}},
+      pos = 12,
+    }}}
   end)
-end)
+end
+
+-- the representation of cmdline and cmdline_block contents changed with ext_newgrid
+-- (which uses indexed highlights) so make sure to test both
+describe('external cmdline', function() test_cmdline(true) end)
+describe('external cmdline (legacy highlights)', function() test_cmdline(false) end)
```

Comment:
Any objection to the `ui/ext_foo` convention for these test groups?

    describe('ui/ext_cmdline', function() test_cmdline(true) end)
    describe('ui/ext_cmdline (legacy highlights)', function() test_cmdline(false) end)

Reply:
none.